node/impl/full/eth.go

```diff
@@ -1692,7 +1698,7 @@ func (e *EthEventHandler) ethGetEventsForFilter(ctx context.Context, filterSpec
 	}
 
 	// Create a temporary filter
-	f, err := e.EventFilterManager.Install(ctx, pf.minHeight, pf.maxHeight, pf.tipsetCid, pf.addresses, pf.keys, true)
+	f, err := e.EventFilterManager.Install(ctx, pf.minHeight, pf.maxHeight, pf.tipsetCid, pf.addresses, pf.keys, false)
 	if err != nil {
```

Comment:
@rvagg This should not always be true. If the user asks for events for a tipset that has been reverted, this will erroneously return no logs for it.